lib/bitmap-transform.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function invertColors(obj) {
+  let bufferArray = new Int32Array(obj);
+
+  let invertTransform = bufferArray.reduce(function(acc, ele, index) {
+    if (index < 54) acc.push(ele);
+    if (index >= 54 && index < 1078) {
+      let inverted = 255 - ele;
+      acc.push(inverted);
+    }
+    if (index >= 1078) acc.push(ele);
```

Comment:
@khalidM3 @reganoneill @dkulp23 inverter looks good!  nice job using `reduce` to accumulate an array!!!!